lib/widgets/login.dart

```diff
@@ -223,7 +225,13 @@ class _AddAccountPageState extends State<AddAccountPage> {
                 decoration: InputDecoration(
                   labelText: zulipLocalizations.loginServerUrlInputLabel,
                   errorText: errorText,
-                  helperText: kLayoutPinningHelperText,
+                  helper: InkWell(
```

Comment:
Hmm, awkward that the ink area is really rigidly rectangular:

![image](https://github.com/zulip/zulip-flutter/assets/22248748/f58ecb17-0d7f-4f29-bccf-a4d503fec667)

For comparison, here's something Google does on a similar text input. This is on web, and I'm hovering on the link:

<img width="481" alt="image" src="https://github.com/zulip/zulip-flutter/assets/22248748/ccaf7aae-2193-4c2e-9631-236c34a4c1b8">

Notice the rounded corners, and that the start of the text is still aligned with the left edge of the text field. I think that alignment would unfortunately be more trouble than it's worth for us to do here.

Probably what I would do here is just not bother with a touch-response effect, and just use a plain `GestureDetector` instead of an `InkWell`. Like we do for links in Zulip message content.

Reply:
Thanks for pointing this out! I agree the current ink rectangle feels a bit rigid.
Following your suggestion, I've replaced `InkWell` with a plain `GestureDetector`.

I have updated the screen recordings in the PR description.